src/libsystemd/sd-bus/busctl.c

```diff
@@ -67,7 +67,7 @@ static usec_t arg_timeout = 0;
 
 static int list_bus_names(sd_bus *bus, char **argv) {
         _cleanup_strv_free_ char **acquired = NULL, **activatable = NULL;
-        _cleanup_free_ char **merged = NULL;
+        _cleanup_free_ /* sic */ char **merged = NULL;
```

Comment:
@keszybz , thanks!

I'm not sure about `/* sic */`.
[`/* strings are not be freed */`](https://github.com/systemd/systemd/blob/29d87223d54fc13e16f444677f0a94ed0755bd88/src/shared/acl-util.c#L228) is better. But the reason is not clear.